src/components/List.js

```diff
@@ -157,17 +157,17 @@ const List = ({ token }) => {
                 <i class="fas fa-cart-plus"></i>
                 Add Item
               </a>
-              <label htmlFor="search-bar">Filter Items</label>
-              <br />
-              <input
-                type="text"
-                name="search-bar"
-                id="search-bar"
-                placeholder="Start typing here..."
-                value={searchInput}
-                onChange={handleSearchChange}
-              />
-              <input type="reset" onClick={handleClear} />
+              <Box display="flex" alignItems="center">
+                <TextField
+                  name="search-bar"
+                  id="search-bar"
+                  label="Start typing here..."
```

Comment:
Would it be better to label the text input so the user knows its a search bar, like "filter items" or "search". Unless we are adding the magnifying icon to the text input, right now it is a bit ambiguous as to what the text input is for. I mean we know because we created it, but would a user know that this is to filter list items?

Reply:
@DeeDowns great suggestion. I was thinking it should be named something different. I think changing the placeholder to `filter list` makes more sense too